temporalcli/commands.workflow_view.go

```diff
@@ -301,11 +305,44 @@ func (c *TemporalWorkflowDescribeCommand) run(cctx *CommandContext, args []strin
 				acts[i].LastFailure = a.LastFailure
 				acts[i].LastWorkerIdentity = a.LastWorkerIdentity
 				acts[i].LastHeartbeatDetails = a.HeartbeatDetails.GetPayloads()
+				acts[i].Paused = a.Paused
+
+				if pauseInfo := a.GetPauseInfo(); pauseInfo != nil {
+					acts[i].PauseTime = timestampToTime(pauseInfo.GetPauseTime())
+
+					switch pausedBy := pauseInfo.GetPausedBy().(type) {
+					case *workflow.PendingActivityInfo_PauseInfo_Manual_:
+						acts[i].PausedBy = pausedBy.Manual.Identity
+					case *workflow.PendingActivityInfo_PauseInfo_Rule_:
+						acts[i].PausedBy = pausedBy.Rule.Identity
+					}
+				}
 			}
 			_ = cctx.Printer.PrintStructured(acts, printer.StructuredOptions{})
 			cctx.Printer.Println()
 		}
 
+		if pauseInfo := resp.GetWorkflowPauseInfo(); pauseInfo != nil {
+			cctx.Printer.Println(color.MagentaString("Paused Activities: %v", len(pauseInfo.GetActivityPauseInfos())))
```

Comment:
Mentioned in other context, but this isn't actually "paused activities", that is shown in "pending activities that are paused" above, this is "settings for future activity pausing".